src/DistortableCollection.js

```diff
@@ -218,11 +218,35 @@ L.DistortableCollection = L.FeatureGroup.extend({
 
     json.images = json.images.reverse();
     json.avg_cm_per_pixel = this._getAvgCmPerPixel(json.images);
-
+    var jsonImages = json.images;
+    savetoLocalStorage(jsonImages);
     return json;
   },
 });
 
 L.distortableCollection = function(id, options) {
   return new L.DistortableCollection(id, options);
 };
+
+function savetoLocalStorage(jsonImages) {
```

Comment:
Let's address this in https://github.com/publiclab/Leaflet.DistortableImage/pull/1225, not here, to keep things separate!